app/databrowser/src/main/java/org/csstudio/trends/databrowser3/model/PVItem.java

```diff
@@ -388,6 +398,13 @@ private void updateUnits(final VType value)
         setUnits(display.getUnit());
     }
 
+    public void updateDescription(final VType value)
+    {
+        final Display display = Display.displayOf(value);
+        if (display.getDescription() != null)
```

Comment:
Can you verify with "ca://..." PVs? Maybe also "jackie://.."?
Not sure if the description will then be `null` or `""`, an empty string.
Maybe this should just in case check for both `null` or `isBlank()`:

```
if (display.getDescription() != null  &&  !display.getDescription().isBlank())
```

Reply:
I've checked with `ca://` and `jackier://`, and the `display.getDescription()` function always returns `null`, whether there's a description or not.

If we want the description to work with both `ca://` and `jackier://`, we could perhaps take inspiration from what's done in `PVTableItem.java` ? :

```java
        if (Settings.show_description &&
            ! (name.startsWith("sim:") ||
               name.startsWith("loc:")))
        {
            // Determine DESC field.
            // If name already includes a field,
            // replace it with DESC field.
            final int sep = name.lastIndexOf('.');
            final String desc_name = sep >= 0
                    ? name.substring(0, sep) + ".DESC"
                    : name + ".DESC";
            try
            {
                final PV new_desc_pv = PVPool.getPV(desc_name);
                desc_flow = new_desc_pv.onValueEvent()
                                       .throttleLatest(Settings.max_update_period_ms, TimeUnit.MILLISECONDS)
                                       .subscribe(value ->
                {
                    if (value instanceof VString)
                        desc_value = ((VString) value).getValue();
                    else
                        desc_value = "";
                    listener.tableItemChanged(PVTableItem.this);
                });
                desc_pv.set(new_desc_pv);
            }
            catch (Exception ex)
            {
                logger.log(Level.WARNING, "Skipping " + desc_name);
            }
        }
    }
```
What do you think?

Reply:
I agree with you, it's better to make do with what's natively possible with PVA than to try to make up for a lack in CA. This would make the code much more complex and could cause bugs later on.